components/chat_widget/src/components/ocs-chat/ocs-chat.css

```diff
@@ -193,6 +212,26 @@
   --code-text-assistant-color: var(--message-assistant-text-color);
   --code-border-assistant-color: color-mix(in srgb, var(--message-assistant-bg-color) 90%, black 10%);;
 
+  --confirmation-overlay-bg-color: rgba(0, 0, 0, 0.5);
+  --confirmation-dialog-bg-color: var(--chat-window-bg-color);
+  --confirmation-dialog-border-color: var(--chat-window-border-color);
+  --confirmation-dialog-shadow-color: var(--chat-window-shadow-color);
+  --confirmation-dialog-border-radius: 0.75em;
+  --confirmation-title-color: var(--input-text-color);
+  --confirmation-title-font-size: 1.125em;
+  --confirmation-message-color: var(--loading-text-color);
+  --confirmation-message-font-size: var(--chat-window-font-size);
+  --confirmation-button-cancel-bg-color: var(--button-background-color-hover);
+  --confirmation-button-cancel-bg-hover-color: var(--message-assistant-bg-color);
```

Comment:
I don't think this should be linked with the `--message-assistant-bg-color`. Rather just use the same raw value.